src/docs/markdown/caddyfile/directives/handle_errors.md

```diff
@@ -43,11 +43,10 @@ handle_errors {
 }
 ```
 
-Reverse proxy to a professional server that is highly qualified for handling HTTP errors and improving your day:
+Simply use [`respond`](/docs/caddyfile/directives/respond) to return the error code and name
 
 ```caddy
 handle_errors {
-	rewrite * /{http.error.status_code}
-	reverse_proxy https://http.cat
+    respond "{http.error.status_code} {http.error.status_text}"
```

Comment:
The problem is that `http.cat` expects the Host to match. Adding `header_up` fixes the problem.

```suggestion
	rewrite * /{http.error.status_code}
	reverse_proxy https://http.cat {
		header_up Host {http.reverse_proxy.upstream.hostport}
	}
```

Reply:
That said, your example is good too, so if you'd like to re-arrange it to fix the `http.cat` bug, and add yours, that'd be great!